### PR TITLE
Add ARIA label field to Hyperlink atom

### PR DESCRIPTION
**Pull request by Scotchester**
One of the things we learned in the course of the accessibility mini-project is that we can use the `aria-label` attribute on `<a>` elements to provide additional context to screen reader users in cases where the cleared link text was something like "Learn more" or when including the context would significantly harm the experience for sighted users (e.g., [repetitive links in a long table](https://www.consumerfinance.gov/policy-compliance/guidance/supervision-examinations/#anchor_manual)).

This PR adds an ARIA label field to the Hyperlink atom that's found in many modules so that Wagtail editors can add a value for the `aria-label` attribute when appropriate.

## Additions

- New ARIA label field on Hyperlinks and the template update to output it

## Testing

1. Pull branch
1. Migrate
1. Add/edit any module that includes a Hyperlink
1. Preview it without adding anything to the ARIA label field and see that it has no `aria-label` attribute
1. Enter something in the ARIA label field and preview it
1. Observe that it now has an `aria-label` attribute

## Screenshots

screen shot 2018-05-07 at 11 59 15

## Notes

- Unfortunately, this doesn't do anything for links added within a rich text field. After the move to Wagtail 2.0, we should consider a Draftail plugin that would enable that.

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Internet Explorer 8, 9, 10, and 11
- [ ] Edge
- [ ] iOS Safari
- [ ] Chrome for Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: cfgov/ask_cfpb/migrations/0015_add_aria-label_to_hyperlink.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
...re.blocks.CharBlock(help_text=b'(Optional) Loads text components in the specified language, if other than English. E.g., use "es"  for Spanish. See https://dev.twitter.com/web/overview/languages for a list of supported language codes.', required=False)), (b'email_title', wagtail.wagtailcore.blocks.CharBlock(help_text=b'(Optional) Custom subject for email shares. If blank, will default to value of blurb field above.', required=False)), (b'email_text', wagtail.wagtailcore.blocks.CharBlock(help_text=b'(Optional) Custom text for email shares. If blank, will default to "Check out this page from the CFPB".', required=False)), (b'email_signature', wagtail.wagtailcore.blocks.CharBlock(help_text=b'(Optional) Adds a custom signature line to email shares. ', required=False)), (b'linkedin_title', wagtail.wagtailcore.blocks.CharBlock(help_text=b'(Optional) Custom title for LinkedIn shares. If blank, will default to value of blurb field above.', required=False)), (b'linkedin_text', wagtail.wagtailcore.blocks.CharBlock(help_text=b'(Optional) Custom text for LinkedIn shares.', required=False))])), ('reusable_text', v1.blocks.ReusableTextChooserBlock(v1.models.snippets.ReusableText))], blank=True),
```
Comment (jimmynotjim) [inline cfgov/ask_cfpb/migrations/0015_add_aria-label_to_hyperlink.py]: Can this be broken down on to new lines?

**Comment by anselmbradford** [comment]
As mentioned in chat, I have the same issue as @jimmynotjim

**Comment by Scotchester** [comment]
Hmm... I'll look into it.

**Comment by Scotchester** [comment]
Seems like I overestimated how much the Hyperlink template was being used. I'll update all the components I can find to use it.

**Comment by marteki** [comment]
@Scotchester what's the status of this PR?

**Comment by anselmbradford** [comment]
SqISRn1GRkddIcc3rG

**Comment by anselmbradford** [comment]
Covered by https://github.com/cfpb/consumerfinance.gov/pull/6497

**Pull request closed without merging**